DuckDuckGo/MainViewController.swift

```diff
@@ -872,18 +876,29 @@ class MainViewController: UIViewController {
         }
     }
 
-    private func addToView(tab: TabViewController) {
+    private func addToWebViewContainer(tab: TabViewController) {
         removeHomeScreen()
         updateFindInPage()
         currentTab?.progressWorker.progressBar = nil
         currentTab?.chromeDelegate = nil
-        addToView(controller: tab)
+        currentTab?.webView.scrollView.contentInsetAdjustmentBehavior = .never
```

Comment:
iOS does weird stuff if this not disabled, once you start messing with the content insets